compiler/damlc/tests/daml-test-files/KeyNotVisibleStakeholders.daml

```diff
@@ -1,7 +1,7 @@
--- @ERROR Stakeholders: 's'
--- @ERROR Stakeholders: 's'
--- @ERROR Stakeholders: 's'
--- @ERROR Stakeholders: 's'
+-- @ERROR 's'
```

Comment:
A bit strange. 
The test was arguably a bit obscure before your change, but this change make it event more.

Reply:
So there were actually two printers for `FailedAuthorizations` which existsing in the code. One in `Pretty.scala` and the other in `Blinding.scala`. It looked like the one in `Blinding` was more user facing -- there was at least one testcase which expected the message it produced -- and so I picked this to be _the_ printer; it was no longer needed in `Blinding`, so I moved it to `Pretty` to replace the original one that was there.

The old errors expected by this `KeyNotVisibleStakeholders.daml` testcase (`Stakeholders: 's'`) came from the original printer in `Pretty.scala`. But it think the message is better now: (`failed due to a missing authorization from 's'`)

The reason for being so cryptic in the `@ERROR` expectation of the testcase is because I wanted the test to work for both new and old versions - I was worried about breaking the compatibility tests -- but maybe I misunderstand how that works.

Should/can we just write the better error message in the testcase?
```
-- @ERROR failed due to a missing authorization from 's'
```

Reply:
>Should/can we just write the better error message in the testcase?
>
>  ``-- @ERROR failed due to a missing authorization from 's'``

I will be very happy with this change.

Reply:
done